index.css

```diff
@@ -0,0 +1,247 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  background-color: #eceef1;
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 0.9em;
+}
+
+/* == header styles == */
+header {
+  padding: 0.4em;
+}
+
+nav {
+  display: flex;
+  flex-direction: row-reverse;
+}
+
+header nav .description {
+  display: inline-block;
+  font-size: 0.75em;
+  vertical-align: middle;
+  padding: 0.1em;
+}
+
+header nav button {
+  background-color: #0077c5;
+  display: inline-block;
+  border: 1px solid #222;
+  margin-right: 3em;
+  font-size: 14px;
+  color: #fff;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Avoid using the same rule many times. Try this

```
.color-white{
  color:#fff;
}
```

In the HMTL file in every element that uses white color

`<div class="color-white"></div>`

- Please once you create the class, delete these rules in the entire CSS file


###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_